Bottom: 4584dc90fb90d84f50a233ca76127ac63112a35c
Top:    6d304d06ead6d0390d8b24ad865c256ced3ddff9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-25 01:49:41 +0100

Refresh of try-to-make-gdk-and-gl-version

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 01d3f46..3041491 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -17,15 +17,9 @@
 #include "crosshair.h"
 #include "error.h"
 #include "../hidint.h"
+#include "gui.h"
 #include "hid/common/draw_helpers.h"
 
-#ifdef ENABLE_GL
-#  include <gtk/gtkgl.h>
-#  include <GL/gl.h>
-#  include "hid/common/hidgl.h"
-#endif
-
-#include "gui.h"
 
 #if !GTK_CHECK_VERSION(2,8,0) && defined(HAVE_GDK_GDKX_H)
 #include <gdk/gdkx.h>
